### PR TITLE
[bitnami/external-dns] Support RFC2136 minTTL

### PR DESCRIPTION
**Pull request by joancafom**
Add support for new argument [introduced](https://github.com/kubernetes-sigs/external-dns/blob/master/CHANGELOG.md#v070---2020-03-10) in v0.7.0 that let the user define a minimal TTL time for records.

Signed-off-by: joancafom <jcarmona@bitnami.com>

**Description of the change**
A new argument specifying was added to the rest of the existing RFC2136 group of args.

**Benefits**
The end user can define the minimal TTL time for records.

**Possible drawbacks**
None

**Applicable issues**

  - fixes bitnami/bitnami-docker-external-dns/issues/5

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/external-dns/values-production.yaml [hunk 1]
```diff
@@ -300,6 +300,7 @@ rfc2136:
   tsigSecretAlg: hmac-sha256
   tsigKeyname: externaldns-key
   tsigAxfr: true
+  minTTL: "0s"
```
Comment (dani8art) [inline bitnami/external-dns/values-production.yaml:303]: I would say the unit is optional [here](https://github.com/kubernetes-sigs/external-dns/blob/master/docs/ttl.md#configure-dns-record-ttl-time-to-live) and it is seconds by default, so you can avoid it. Also, I would add some docs about `time.Duration` and `time.ParseDuration` such as possible values.

```yaml
  ## Default unit `s`. Possible values [ns, us, ms, s, m, h], see more https://golang.org/pkg/time/#ParseDuration
  minTTL: "0"
```

File: bitnami/external-dns/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: external-dns
-version: 3.0.2
+version: 3.0.3
```
[the remark this answers is not in the document]
Comment (joancafom) [reply bitnami/external-dns/Chart.yaml:3]: That is true! Thanks!

File: bitnami/external-dns/values-production.yaml (hunk 1 above)
Comment (joancafom) [reply bitnami/external-dns/values-production.yaml:303]: Initially, I implemented it without the unit, but errors show up. I will give it a try and post here the results. About the docs, that's a nice idea!

Comment (joancafom) [reply bitnami/external-dns/values-production.yaml:303]: This is the outcome you get when no units is provided (using a magnitude of 5):
```console
time="2020-05-27T08:11:10Z" level=fatal msg="flag parsing error: time: missing unit in duration 5"
```

Comment (dani8art) [reply bitnami/external-dns/values-production.yaml:304]: Cool, it seems they have implemented a validation in their side wrapping the original library, so let's keep the unit!

**Pull request merged**